lib/OpenQA/WebAPI.pm

```diff
@@ -393,6 +396,10 @@ sub startup {
     $api_public_r->get('job_templates/:job_template_id')->name('apiv1_job_template')->to('job_template#list');
     $api_ra->delete('job_templates/:job_template_id')->to('job_template#destroy');
 
+    # api/v1/job_templates_scheduling
+    $api_public_r->get('experimental/job_templates_scheduling/:id')->name('apiv1_job_templates_schedules')
```

Comment:
If the id is numeric you could use `$api_public_r->get('experimental/job_templates_scheduling/<id:num>')` here.